License headers:

tools/testing/selftests/futex/functional/Makefile:
# SPDX-License-Identifier: GPL-2.0

--- a/tools/testing/selftests/futex/functional/Makefile.orig	2021-09-23 00:08:00.277234760 -0700
+++ b/tools/testing/selftests/futex/functional/Makefile	2021-09-23 00:08:47.545236318 -0700
@@ -20,7 +20,8 @@ TEST_GEN_FILES := \
 	futex_wait \
 	futex_requeue \
 	futex_wait_multiple \
-	futex2_wait
+	futex2_wait \
+	futex2_waitv
 
 TEST_PROGS := run.sh
 
